R/pkg/NAMESPACE

```diff
@@ -6,10 +6,15 @@ importFrom(methods, setGeneric, setMethod, setOldClass)
 #useDynLib(SparkR, stringHashCode)
 
 # S3 methods exported
+export("sparkR.session.getOrCreate")
```

Comment:
I agree - I think it's important to have session in there and thought it would better to be more explicit about the getOrCreate behavior but as commented that name was unusual in R.